pkg/cri/opts/spec_windows_opts.go

```diff
@@ -67,9 +67,10 @@ func parseMount(osi osinterface.OS, mount *runtime.Mount) (*runtimespec.Mount, e
 			}
 		}
 		var err error
+		originalSrc := src
 		src, err = osi.ResolveSymbolicLink(src)
 		if err != nil {
-			return nil, fmt.Errorf("failed to resolve symlink %q: %w", src, err)
+			return nil, fmt.Errorf("failed to resolve symlink '%q' (resolved to '%q'): %w", originalSrc, src, err)
```

Comment:
The `resolved to '%q'` bit might not be of much use (unless things are different on Windows somewhere, didn't see from looking) as it seems like for Unix (and just a general Go convention) they return an empty string if there's an error.

Reply:
also we don't have to put `'` as `%q` already wraps a string with `"`

Reply:
Should be updated now.